docs/docsite/rst/playbooks_reuse_roles.rst

```diff
@@ -170,14 +170,14 @@ Or, again, using the newer syntax::
 
     - hosts: webservers
       tasks:
-      - import_role: 
+      - import_role:
           name: foo
         tags:
         - bar
         - baz
 
 .. note::
-    This *tags all of the tasks in that role with the tags specified*, overriding any tags that are specified inside the role. If you find yourself building a role with lots of tags and you want to call subsets of the role at different times, you should consider just splitting that role into multiple roles.
+    This *tags all of the tasks in that role with the tags specified*, appending to tags that are specified inside the role. If you find yourself building a role with lots of tags and you want to call subsets of the role at different times, you should consider just splitting that role into multiple roles.
```

Comment:
Not sure exactly what you mean by import vs include. So it's true for `roles` but not `include_role`?